app/resources/api/v2/sample_resource.rb

```diff
@@ -14,6 +14,7 @@ class SampleResource < BaseResource
       attribute :uuid
       attribute :control
       attribute :control_type
+      attribute :supplier
```

Comment:
This attribute is dependent on both sample manifest and supplier. This means that for every sample, we'll generate two additional queries, which results in significant slowdown if retrieving large numbers of samples.

If you're going to add methods to the API that depend on other records, then you *need* to ensure that they are added to the default includes for that resource, otherwise you'll bog down all requests for sample.

Either that, or expose the actual supplier object, that way we could only load it when we actually needed it.